contract/RadicleCloudBase.sol

```diff
@@ -0,0 +1,191 @@
+// SPDX-License-Identifier: GPL-3.0
+
```

Comment:
I'm missing a few functions I think:
1. A function for the operator to cancel the subscription and return all unspent deposited tokens to the user (eg. in case of abuse.
2. A function for the user to see how much money they have in the contract that they can withdraw
3. A function for the user to withdraw all (the maximum) amount that hasn't been spent, from the contract.

Reply:
2 is missing, but for 1 & 3:
https://github.com/radicle-dev/radicle-cloud/pull/1/commits/36dcf9dff66ea8ad4d06520d100d38192e699a9f#diff-f9840e5b47321a0c9bbf85905168b75666bf3d8b6873f0cd476bbf4d3d23d2e4R212-R239